python/semantic_kernel/connectors/ai/mistral_ai/services/mistral_ai_chat_completion.py

```diff
@@ -95,20 +96,22 @@ def __init__(
     async def get_chat_message_contents(
         self,
         chat_history: "ChatHistory",
-        settings: "MistralAIChatPromptExecutionSettings",  # type: ignore[override]
+        settings: "PromptExecutionSettings",
         **kwargs: Any,
     ) -> list["ChatMessageContent"]: 
         """Executes a chat completion request and returns the result.
 
         Args:
             chat_history (ChatHistory): The chat history to use for the chat completion.
-            settings (MistralAIChatPromptExecutionSettings): The settings to use
+            settings (PromptExecutionSettings): The settings to use
                 for the chat completion request.
             kwargs (Dict[str, Any]): The optional arguments.
 
         Returns:
             List[ChatMessageContent]: The completion result(s).
         """
+        settings = self.get_prompt_execution_settings_from_settings(settings)
```

Comment:
Mypy doesn't complain about settings not being of type `MistralAIChatPromptExecutionSettings`?

Reply:
Good call, i had something wrongly configured locally with mypy, i needed the assert Statement !